composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-mbstring": "*",
         "behat/gherkin": "^4.5.1",
         "behat/transliterator": "^1.2",
-        "symfony/console": "~2.5||~3.0||~4.0",
+        "symfony/console": "~2.7.40||^2.8.33||~3.3.15||^3.4.3||^4.0.3",
```

Comment:
as this bumps the min version to 2.7+, you could also clean `InteractiveContextIdentifier::interactionIsNotSupported`

Reply:
We should probably discuss whether it's ok to drop 2.5 and 2.6

Reply:
I might do a separate PR to do this cleanup and more.

Reply:
@ciaranmcnulty didn't see your message, but I think this is kind of a no-brainer, isn't it?

Reply:
There's no explicit BC policy in place; I think @everzet's opinion matters here

Reply:
I think a sensible policy might be to forbid dropping minor versions of sf unless they are no longer supported.

Reply:
2.5 and 2.6 are EOLed since years. So that's fine here.